src/main/java/de/tum/in/www1/artemis/web/rest/ProgrammingSubmissionResultSimulationResource.java

```diff
@@ -0,0 +1,95 @@
+package de.tum.in.www1.artemis.web.rest;
+
+import java.util.Optional;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.http.ResponseEntity;
+import org.springframework.security.access.prepost.PreAuthorize;
+import org.springframework.transaction.annotation.Transactional;
+import org.springframework.web.bind.annotation.*;
+
+import de.tum.in.www1.artemis.domain.*;
+import de.tum.in.www1.artemis.domain.participation.*;
+import de.tum.in.www1.artemis.service.*;
+import de.tum.in.www1.artemis.service.ProgrammingSubmissionResultSimulationService;
+
+@RestController
+@RequestMapping("/api")
+@Transactional
+public class ProgrammingSubmissionResultSimulationResource {
```

Comment:
this should only be available in DEV mode, please use a profile here

Reply:
I added the profile